### ISSUE TITLE
Can't run offline

### ISSUE THREAD

**Issue by mikerossiter** [ohmplatform/FreedomGPT#121]
When I go into airplane mode and run yarn start from root I get this error in the logs in the terminal.

 [TypeError: fetch failed] {
  cause:  [Error: getaddrinfo ENOTFOUND api.openai.com] {
  errno: -3008,
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'api.openai.com'
}

I can see it's moaning about not being able to hit the api.openai.com endpoint so can I turn this off somehow? I don't need chatgpt anyway.

**Comment by WinterAtne** [comment]
I just learned about this repo today and haven't looked at the code to much but that sounds correct. This project relies on openai to run, so accessing their api sounds necessary to running the program. (please correct me If I am wrong)

**Comment by mikerossiter** [comment]
Yes it seems that way. The app itself though says this when you go ConnectAI:
"Edge AI is AI that runs locally on your machine. The data never leaves your device, so everything stays private. You can even use this AI when you are completely offline, on a plane, or a deserted island."

**Comment by reconSuave** [comment]
FreedomGPT ▶︎ renderer ▶︎ utils ▶︎ app ▶︎ const.ts

> ✘ remove unused constant OPENAI_API_HOST

![how to fix le ape](https://i.imgur.com/O84nPje.png)
